src/components/common/CapitalLink.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <component :is="element" class="capital-link">{{ text }}</component>
```

Comment:
Why not just have a `<slot>` here? This way we could be much more adaptable if someone would, let's say want to add an icon or more complicated content 😄 
This would also make using such component much more natural.

```suggestion
  <component :is="element" class="capital-link">
    <slot />
  </component>
```